client/src/scenes/SinglePlan.jsx

```diff
@@ -133,12 +133,12 @@ class SinglePlan extends Component {
             });
 
         return (
-            <Wrapper me={me}>
+            <Wrapper>
                 {plan.PL_NAME && (
                     <div className="container">
                         <div className="container">
                             <h1>{plan.PL_NAME}</h1>
-                            {!me && (
+                            {!true && (
```

Comment:
this would make this message never appear.
while this will be irrelevant in a bit with the new plan page, is there an easy way to tell here if the user is signed in or not?

Reply:
this will require migrating the page to hooks. I would pass on that atm - we are almost done with the new implemntation